.final_status

```diff
@@ -0,0 +1 @@
+SUCCESS
```

Comment:
If you don't mind I rather not commit this file on the repo and the we create it inside the ci pipeline with something like `echo SUCCESS .final_status`